public/src/app/issue-list/issue-list.component.ts

```diff
@@ -33,7 +33,10 @@ export class IssueListComponent implements OnInit {
       this.issues.reduce(
         (text, node) => {
           const repo = repoShortNameFromGithubUrl(node.url);
-          return text + `<li>${escapeHtml(node.title)} (<a href='${node.url}'>${repo}#${node.number}</a>)</li>\n`
+          const check = node.timelineItems && node.timelineItems.nodes.length ? '✔ ' : '';
+          const prs = check ? node.timelineItems.nodes.reduce(
+            (current, pr) => `${current} <a href='${pr.subject.url}'>#${pr.subject.number}</a>`, ' 🔗 ') : '';
```

Comment:
```suggestion
          const prs = check ?
            node.timelineItems.nodes.reduce((current, pr) => `${current} <a href='${pr.subject.url}'>#${pr.subject.number}</a>`, ' 🔗 ') :
            '';
```

Reply:
I find the current wrapping confusing - took me a bit to see that the first part of the ternary operator was in the first line...